battle/main.py

```diff
@@ -4772,8 +4772,11 @@ def calm_lands_manip():
         calm_lands_gems()
     else:
         print("++++ Gems good. NEA manip logic.")
-        advancePreX, advancePostX = rngTrack.nea_track()  # returns integers
-        if advancePreX not in [0,2] and advancePostX not in [0,2]:  # Non-zero for both
+        advancePreX, advancePostX = rng_track.nea_track()  # returns integers
+        if advancePreX not in [0, 2] and advancePostX not in [
+            0,
+            2,
+        ]:  # Non-zero for both
```

Comment:
If we pull this comment up to above the `if` statement, my money says that this will all fit on one line again without black doing it's thing.